experimental/segmenter/src/line.rs

```diff
@@ -99,10 +100,22 @@ impl Default for LineBreakOptions {
 pub struct LineBreakSegmenter {
     options: LineBreakOptions,
     payload: DataPayload<LineBreakDataV1Marker>,
-    dictionary_payload: DataPayload<UCharDictionaryBreakDataV1Marker>,
+    dictionary_km_payload: Option<DataPayload<UCharDictionaryBreakDataV1Marker>>,
```

Comment:
Thought: would it make sense to use the `Dictionaries` structure here? You'd have to make it own its data, but you're passing it by reference anyway so there's no cloning.